recipes/equilibrator-cache/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "equilibrator-cache" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8a4326051c5c4171167c24ee050892f4060e83ff8ca5ef75469828dd5b09f564
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --ignore-installed"
+
+requirements:
+  build:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - importlib_resources >=3.0.0
+    - numpy >=1.19.0
+    - pandas >=1.1.0
+    - python-levenshtein >=0.12.0
+    - scipy >=1.5.0
+    - sqlalchemy >=1.3.0
+    - quilt <3.0
```

Comment:
Lowest version of `quilt` on conda-forge is 3.0.5. If it needs to be `<3.0` then you need to raise an issue at https://github.com/conda-forge/quilt-feedstock to get an older version built.